src/components/Settings/System/Auth/ExternalSAML.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <b-form-group
+    :label="title"
+    label-size="lg"
+  >
+    <b-form-group label-cols="2">
+      <b-form-checkbox
+        v-model="value.enabled"
+        :value="true"
+        :unchecked-value="false"
+      >
+        {{ $t('provider-enabled') }}
+      </b-form-checkbox>
+    </b-form-group>
+    <b-form-group
+      v-if="value.enabled"
+      :label="$t('saml.cert')"
+      label-cols="2"
+      :description="$t('saml.desc.cert')"
+    >
+      <b-input-group>
+        <b-form-textarea
+          v-model.trim="value.cert"
+        />
+      </b-input-group>
+    </b-form-group>
+    <b-form-group
+      v-if="value.enabled"
```

Comment:
You could group all v-if="value.enabled" under a single div.  So only one v-if is needed.
Otherwise LGTM